httpbin/handlers.go

```diff
@@ -740,19 +737,33 @@ func handleBytes(w http.ResponseWriter, r *http.Request, streaming bool) {
 	rng := rand.New(src)
 
 	w.Header().Set("Content-Type", "application/octet-stream")
+	w.Header().Set("Content-Length", strconv.FormatInt(numBytes, 10))
+
 	w.WriteHeader(http.StatusOK)
 
+	// prepare a buffer of chunkSize bytes
+	var i int64
 	var chunk []byte
-	for i := 0; i < numBytes; i++ {
+	for i = 0; i < chunkSize; i++ {
 		chunk = append(chunk, byte(rng.Intn(256)))
-		if len(chunk) == chunkSize {
-			write(chunk)
-			chunk = nil
-		}
 	}
-	if len(chunk) > 0 {
-		write(chunk)
+
+	f := w.(http.Flusher)
+	// send the buffer as many times as needed
+	var numChunk = numBytes / chunkSize
+	// fmt.Println("numChunk :", numChunk)
+	for i = 0; i < numChunk; i++ {
+		w.Write(chunk)
```

Comment:
WIth this change, it appears we now generate only chunkSize random bytes and write those same bytes to the response for every chunk.  Let's maintain the original behavior, where numBytes random bytes are generated and written to the response.

Reply:
The whole point for me is to have a base code for performance measurement. Randomizing every bytes just kill the performance on very high speed network and render this project useless for me.

I started this because I need a http server to get/post for performance measurements. Instead of starting from scratch I noticed your project and decided to try building upon. That the whole point of this PR, to see if our goal are compatible or not.

I think now it's not the good approach. Since your projet can be used as a lib, I think it's better for me to build my own project and 'speed measurement' endpoints and add your endpoints as a lib (if possible).

You can forget this PR then. Sorry about that.

Reply:
> I started this because I need a http server to get/post for performance measurements. Instead of starting from scratch I noticed your project and decided to try building upon. That the whole point of this PR, to see if our goal are compatible or not.

Gotcha!  Peak performance has never been a specific goal for this project, but I would not be opposed to adding a new endpoint to stream an arbitrary number of fixed bytes.  I could even talk myself into it being compatible with this /stream-bytes endpoint as an additional `?bytes=<specific chunk to stream back>` parameter to allow the client to control the actual bytes that will be streamed.

> I think now it's not the good approach. Since your projet can be used as a lib, I think it's better for me to build my own project and 'speed measurement' endpoints and add your endpoints as a lib (if possible).

That approach also seems pretty reasonable.  If you do go down this road, please let me know if you run into any trouble "embedding" go-httpbin's endpoints into your more purpose-built server, I'd love to know where the pain points are and whether they can be addressed.

> You can forget this PR then. Sorry about that.

No worries, thanks for opening it!